### PR TITLE
Updating PR Guidelines

### PR DESCRIPTION
**Pull request by lalitb**
Trying to update existing PR guidelines to ensure smoother ( and probably faster ) merge process:

 - With current small number of Approvers ( and their distribution across different companies ), making number of required approvals ( from different company ) as 1 ( from existing 2 ). We can review this after course of time once we have good distribution.
- Requirement for Updating the changelog (CHANGELOG.md) for non-trivial changes.
- Handling lots of discussions in given PR.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (ThomsonTan) [inline CONTRIBUTING.md:102]:
```suggestion
* It has received at least one approval from [Approvers](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver)
```

Comment (ThomsonTan) [reply CONTRIBUTING.md:102]: Also should we require at least two approvals? Perhaps one approval for trivial change is fine.

Comment (lalitb) [reply CONTRIBUTING.md]: Good point. Let's hear from others before finalizing it.

File: CONTRIBUTING.md
Comment (jsuereth) [inline CONTRIBUTING.md:104]: We discussed in the SiG, and I thought we had said: "Approval from Approvers/Mainters where the approver is from a different company".   Can we update this to denote that?

**Review by jsuereth — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md
Comment (ThomsonTan) [reply CONTRIBUTING.md:104]: Is this already covered by line 104? But I agree we need change the word a little bit.

Comment (lalitb) [reply CONTRIBUTING.md]: Yes, it's already there. But have now reworded slightly, let me know if it still not clear.

File: CONTRIBUTING.md
Comment (lalitb) [reply CONTRIBUTING.md]: OK, I have updated guideline to have
 - One approval ( from different company ) for PRs raised by approver / maintainers.
- Two approvals with one approval from different company for all other PRs.
Let me know if that will work ?

**Review by ThomsonTan — APPROVED** [review]

**Review by jsuereth — APPROVED** [review]

**Pull request merged**
